pseudo.md

```diff
@@ -0,0 +1,29 @@
+What I need to get:
+
+Pseudo Code
+
+GET ALL ICE CREAM = "/icecream"
+
+GET ICE CREAM BY FLAVORS = "/icecream/:flavors"
+
+GET ALL ICE CREAM CATEGORIES = "/categories"
+
+GET FRUITY = "/categories/:fruity"
```

Comment:
"/categories/fruity" would be the url in this case. You don't need the ":" because "fruity" is a category. It follows the format "/categories/:flavorCategory"